parser.y

```diff
@@ -686,6 +686,7 @@ import (
 	SignedLiteral			"Literal or NumLiteral with sign"
 	DefaultValueExpr		"DefaultValueExpr(Now or Signed Literal)"
 	NowSymOptionFraction		"NowSym with optional fraction part"
+        CharsetNameOrDefault            "Character set name or default"
```

Comment:
Please reindent this line with tab.

Reply:
sry, I used to set expandtab in vim, and gofmt does not work in *.y file. I'll reindent the tab.